<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>214</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MusicAlarm</string>
  </property>
  <property name="toolTip">
   <string>Wake me up before you go-go...</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
    </font>
   </property>
   <widget class="QLabel" name="counter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>342</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>000:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="dreamProg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>172</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dreamLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Dream</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="correctionMode">
     <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
    </property>
    <property name="displayFormat">
     <string notr="true">yyyy-MM-dd HH:mm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QToolButton" name="getFile">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>27</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="musicSrc">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtTop</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>selectFile()</slot>
  <slot>start()</slot>
 </slots>
</ui>
